docs/reference/server-configuration.md

```diff
@@ -801,7 +923,7 @@ For more info see the WITH clause properties in
 
 ## `ksql.sink.replicas` (Deprecated)
 
-**Per query:** yes
+**Per query:** yes (for {{ site.ccloud }}, settable with ALTER SYSTEM only)
```

Comment:
@rodesai to sanity check but I'm guessing it's definitely not. Given that it's deprecated I also wouldn't bother adding it to the list of system properties that can be modified via `ALTER SYSTEM` -- seem reasonable?

```suggestion
**Per query:** no
```